buildozer/npm/index.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @fileoverview NodeJS binding to run the buildozer command through a convenient API
+ */
+const {spawnSync} = require('child_process');
+
+/**
+ * Models the data structure of a buildozer command file.
+ * You should group as many commands as possible into one call to `run` for efficiency.
+ * https://github.com/bazelbuild/buildtools/tree/master/buildozer#do-multiple-changes-at-once
+ */
+export type CommandBatch = {
```

Comment:
NIT: shouldn't this be an `interface` since `CommandBatch` is restricted to a object?